### DISCUSSION TITLE
No module named 'chromadb'

### DISCUSSION THREAD

**Discussion by wangquan3294** [vanna-ai/vanna#668] [Q&A] [answered]
i write a code as "from vanna.ollama import Ollama from vanna.chromadb import ChromaDB_VectorStore" but it tells " File "C:\Users\DELL\Desktop\testVanna.py", line 2, in from vanna.chromadb import ChromaDB_VectorStore File "C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb_init_.py", line 1, in from .chromadb_vector import ChromaDB_VectorStore File "C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb\chromadb_vector.py", line 4, in import chromadb ModuleNotFoundError: No module named 'chromadb'" who can help me?

**Comment by MyNameIsNeXTSTEP** [comment]
Any workarounds or solutions on this ?

**Comment by RodolfoMontenegro** [comment, marked as answer]
@wangquan3294 this:
"C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb_init_.py", line 1, in from .chromadb_vector import ChromaDB_VectorStore File "C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb\chromadb_vector.py", line 4, in import chromadb ModuleNotFoundError: No module named 'chromadb'"

probably means you didnt do the propper pip install from vanna, for example 'vanna[postgres, ollama, chromadb]' so the chromadb library from vanna library is not being installed.

I hope that helps!
